docs/book/src/forc/plugins/forc_client/index.md

```diff
@@ -1,26 +1,76 @@
 # `forc-client`
 
-Forc plugin for interacting with a Fuel node.
+Forc plugin for interacting with a Fuel node. Since transactions are going to require some gas, you need to sign them with an account that has enough tokens to pay for them.
 
-## Initializing the wallet and adding accounts
+We offer multiple ways to sign the transaction:
 
-If you don't have an initialized wallet or any account for your wallet you won't be able to sign transactions.
+  1. Sign the transaction via your local wallet using forc-client which integrates with our CLI wallet, forc-wallet.
+  2. Use the default signer to deploy to a local node
+  3. Use forc-wallet to manually sign transactions, and copy the signed transaction back to forc-client.
 
-To create a wallet you can use `forc wallet new`. It will ask you to choose a password to encrypt your wallet. After the initialization is done you will have your mnemonic phrase.
+The easiest and recommended way to interact with deployed networks such as our testnets is option 1, using forc-client to sign your transactions which reads your default forc-wallet vault. For interacting with local node, we recommend using the second option, which leads forc-client to sign transactions with the a private key that comes pre-funded in local environments.
 
-After you have created a wallet, you can derive a new account by running `forc wallet account new`. It will ask your password to decrypt the wallet before deriving an account.
+## Option 1: Sign transactions via forc-client using your local forc-wallet vault
 
-## Signing transactions using `forc-wallet` CLI
+If you used forc-wallet before, a vault which securely holds your private key is created and written to your file-system in a password encrypted format. forc-client is compatible with forc-wallet such that it can read that vault by asking you your password and use your account to sign transactions.
```

Comment:
```suggestion
If you've used `forc-wallet` before, you'll already have a secure, password-protected vault holding your private key written to your file-system. `forc-client` is compatible with `forc-wallet` such that it can read that vault by asking you your password and use your account to sign transactions.
```